docs/index.md

```diff
@@ -21,17 +21,17 @@ Download the latest version of the CLI for your operating system and make sure t
 
     [![Download](https://api.bintray.com/packages/codacy/pulse/event-cli/images/download.svg)](https://bintray.com/codacy/pulse/event-cli/_latestVersion)
 
-1.  Run the command to download the correct binary for your operating system, changing `<VERSION>` to the latest version obtained on the previous step:
-
-    | Operating system | Command to download the binary |
-    | ---------------- | ------------------------------ |
-    | macOS            | `curl -fsSL -o event-cli https://dl.bintray.com/codacy/pulse/event-cli/<VERSION>/pulse-event-cli_darwin_amd64/pulse-event-cli` |
-    | Linux 32-bit     | `curl -fsSL -o event-cli https://dl.bintray.com/codacy/pulse/event-cli/<VERSION>/pulse-event-cli_linux_386/pulse-event-cli` |
-    | Linux 64-bit     | `curl -fsSL -o event-cli https://dl.bintray.com/codacy/pulse/event-cli/<VERSION>/pulse-event-cli_linux_amd64/pulse-event-cli` |
-    | Windows 32-bit   | `curl -fsSL -o event-cli.exe https://dl.bintray.com/codacy/pulse/event-cli/<VERSION>/pulse-event-cli_windows_386/pulse-event-cli.exe` |
+2.  Run the command to download the correct binary for your operating system, changing `<VERSION>` to the latest version obtained on the previous step:
```

Comment:
I think it's better to number all sections as 1. The actual values will always be correct and if you want to reorder items, you can do it without worrying the numbers won't match the new order.

Reply:
I know, but the formatter that we use formats this automatically

Reply:
I agree and also do this when editing the Markdown. However, I still haven't figured out a way to configure [Remark](https://github.com/remarkjs/remark) to avoid rewriting the list item markers as sequential numbers.

Reply:
I'll try to make this work on the codacy/docs repository and will propagate the settings to this repository afterwards:

https://github.com/codacy/docs/pull/298